c2corg_ui/static/js/documentediting.js

```diff
@@ -551,10 +589,23 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
     // init empty conditions_levels for ng-repeat
     outing['locales'][0]['conditions_levels'] = [{'level_snow_height_soft': '', 'level_snow_height_total': '', 'level_comment': '', 'level_place': ''}];
   }
-
   return outing;
 }
 
+/**
+ * @returns {Object} url parameters
+ * @private
+ */
+app.DocumentEditingController.prototype.parseUrl_ = function() {
```

Comment:
it doesn't work for me, it gives back some wierd stuff like `add?docToAssociateId : 736688?doctype=routes`.